### PR TITLE
Keep green

### PR DESCRIPTION
**Pull request by chrisdothtml**

### CODE REVIEW

**Comment by chrisdothtml** [comment]
@KevinGrandon failed CI looks like same issue as other repo?

**Comment by KevinGrandon** [comment]
Yeah looks like we need to update the base image in the Dockerfile here.

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM node:8.11.3@sha256:04986974434fc565529feaac1d62cce4f9fe99ba4906f076ce498000120a45d4
 
+ENV YARN_VERSION 1.13.0
```
Comment (KevinGrandon) [inline Dockerfile]: Hmm, seems like this is not installing the proper yarn version? Maybe we can use the uber-web base image instead of node?

**Review by KevinGrandon — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (chrisdothtml) [reply Dockerfile]: Yeah, would also require changing the ~hard-coded~ node version in this package so wanted to try this first. Will switch it to our base image again

**Review by KevinGrandon — APPROVED** [review]

**Comment by chrisdothtml** [comment]
!merge

**Pull request merged**